internal/presenters/templates/local_finding.tmpl

```diff
@@ -1,15 +1,15 @@
 {{- define "details" }}
 {{- if gt ($.OpenFindings | len) 0 }}{{ "Open Issues" | title }}
 
-        {{- range $finding := $.OpenFindings}}
+        {{- range $finding := ($.OpenFindings | sortFindingBySevAsc)}}
                 {{- renderToString "finding" $finding }}
         {{- end }}
 {{- end }}
-{{- if .ShowDivider}}{{divider}}{{- end}}
-{{- if $.ShowIgnored }}{{"Ignored Issues" | title }}
+{{- if or (and (gt ($.OpenFindings | len) 0) (gt ($.IgnoredFindings | len) 0)) (eq (getValueFromConfig "include-ignores") "true") }}{{divider}}{{- end}}
+{{- if eq (getValueFromConfig "include-ignores") "true" }}{{"Ignored Issues" | title }}
 
 {{- if gt ($.IgnoredFindings | len) 0 }}
-        {{- range $finding := $.IgnoredFindings}}
+        {{- range $finding := ($.IgnoredFindings | sortFindingBySevAsc)}}
```

Comment:
**issue:** `sortFindingBySevAsc` remove use of hard coded value. Severity sort order is provided as part of the `Summary` object

Reply:
Based on https://github.com/snyk/go-application-framework/pull/269/commits/ad55021a94f37c68bc5a5469389c0c4bd64fa96e